docs/dscom.build/ReadMe.md

```diff
@@ -0,0 +1,229 @@
+# dscom.build

```

Comment:
This should be moved to"src/dscom.build"

Reply:
Done in e36ea6731206b710e736dd335e93233de553801f